ci/lib/stage-test-examples.jenkinsfile

```diff
@@ -14,7 +14,8 @@ stage('test-examples') {
                     cd CI-Examples/openjdk
                     if [ "${os_release_id}" != 'ubuntu' ]
                     then
-                        sed -i -e "/file:\\/usr\\/lib\\/jvm\\/java-11-openjdk-amd64/d" java.manifest.template
+                        export jdk_version=$(java -version 2>&1 | head -n 1 | cut -d'"' -f2 | cut -d '.' -f1)
```

Comment:
We have similar code in config-alpine jenkinsfile and config-centos

Reply:
Done